app-localize-behavior.html

```diff
@@ -157,6 +157,15 @@
     },
 
     /**
+     * When there is no translation for a given key,
+     * the key will be used when this is true.
```

Comment:
How about rephrasing to "If true, will use the provided key when the translation does not exist for that key"